core/src/main/scala/org/apache/spark/sql/execution/ObjectHashMapAccessor.scala

```diff
@@ -1311,10 +1309,9 @@ case class ObjectHashMapAccessor(@transient session: SnappySession,
     case StringType if !multiMap =>
       // copy just reference of the object if underlying byte[] is immutable
       val stringVar = resultVar.value
-      val platformClass = classOf[Platform].getName
       val bytes = ctx.freshName("stringBytes")
       s"""byte[] $bytes;
-        if ($stringVar.getBaseOffset() == $platformClass.BYTE_ARRAY_OFFSET
+        if ($stringVar.getBaseOffset() == Platform.BYTE_ARRAY_OFFSET
```

Comment:
No the method is still part of generated code (i.e. it is *not* ${Platform.BYTE_ARRAY_OFFSET}"). The only change here is that explicit qualified platformClass is unnecessary since it in the default imports of Spark's CodeGenerator.